src/components/GoogleSignin/GoogleAuthenticationService.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import axios from 'axios';
+import GoogleAuthenticationDTO from "./GoogleAuthenticationDTO";
+
+class GoogleAuthenticationService {
+
+    constructor() {
+    }
+
+    createSession(response) {
```

Comment:
It's strange to call this method `createSession` when the name of the class has nothing to do with the Session concept. Why not calling it authenticate?
Also, calling the parameter `response` is confusing. What about calling it `googleLoginResponse`?

Reply:
resolved